### PR TITLE
add card component

### PR DESCRIPTION
**Pull request by cezaraugusto**
Added card component previously called panel. there's a component in rewards with the same name so I had to adapt. I like it tho.

Ideally, the rewards components should re-use it so one-change-changes-it-all but left them as-is for now. Welcome page was updated to make use of it.

### CODE REVIEW

**Comment by NejcZdovc** [comment]
I agree with this. I think that we should in most cases only pass pre-defined options in the component and then component can generate style based on that. I think theming should be more about predefined options then on custom css properties. For padding and merging we can always provide wrapper in parent component that is using that component.

I tried to reduce this to minimum in rewards, but still I am passing too much px props in components. I will refactor this in near future. One good example of my abuse is Token. I am passing css for color and font size for 3 different elements. I will group this into 3 types now that I know all variations and will accept only one prop type which would accept strings small | medium | large.

File: src/components/layout/card/style.ts [hunk 1]
```diff
@@ -0,0 +1,22 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License. v. 2.0. If a copy of the MPL was not distributed with this file.
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import styled from 'styled-components'
+import { CardProps } from './index'
+
+const StyledCard = styled<CardProps, 'div'>('div')`
```
Comment (NejcZdovc) [inline src/components/layout/card/style.ts]: nit: can we use `export const StyledCard`

Comment (NejcZdovc) [reply src/components/layout/card/style.ts]: because if we add one more div to it, then default will not work anymore

Comment (cezaraugusto) [reply src/components/layout/card/style.ts]: sure

**Pull request merged**